src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -22,14 +21,13 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
     private _adyenv2?: AdyenV2PaymentInitializeOptions;
     private _adyenPaymentComponent?: AdyenComponent;
     private _adyenCardVerificationComponent?: AdyenComponent;
-    private _adyenComponentState?: AdyenCardState;
+    private _componentState?: ComponentState;
```

Comment:
You can remove the `adyen` prefix for private variables, properties, methods etc... But for exported interface names, as I said earlier, you should keep the prefix.